docs/development.adoc

```diff
@@ -324,7 +324,16 @@ $ TEST_EXEC_NODES=1 make test-source-e2e
 ----
 +
 
-* For the entire integration test suite use:
+* For the login and logout command integration test:
++
+----
+$ make test-odo-login-e2e
+----
++
+
+NOTE: `make test-odo-login-e2e` doesn't honour enviornment variable `TEST_EXEC_NODES`. So by default it runs login and logout command integration test suite on a single ginkgo test node sequentially to avoid race condition in parallel run.
```

Comment:
`environment`

Reply:
`to avoid race conditions in a parallel run`

Reply:
Done